src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -22,43 +40,64 @@ class ResultsList extends Component {
 
   render() {
     const {
+      context,
+      facilityTypeName,
+      inProgress,
+      position,
+      searchString,
       results,
       isMobile,
-      currentQuery,
       pagination: { currentPage, totalPages },
     } = this.props;
 
-    if (currentQuery.inProgress) {
+    if (inProgress) {
       return (
         <div>
-          <LoadingIndicator message="Loading results..." />
+          <LoadingIndicator
+            message={`Searching for ${facilityTypeName}
+            in ${searchString}`}
+            setFocus
+          />
         </div>
       );
     }
 
     if (!results || results.length < 1) {
+      /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
       return (
-        <div className="facility-result">
+        <div
+          className="search-result-title facility-result"
+          tabIndex={0}
```

Comment:
Do we always want these to be in the tab order, or do we just want to be able to focus on them? We've usually tried to do the later, and there's a focus helper function that sets the tabindex to -1 and focuses.

Reply:
I'm using 0 because I think we want users to be able to tab through these informational things. Making an assumption about this: -1 is for modals or things that are not part of the regular flow.

Reply:
Alright Trevor says to use -1. I will refactor this to use the helper.